setup.py

```diff
@@ -46,7 +46,7 @@
     ],
     'test': [
         "hypothesis==3.44.26",
-        "pytest~=3.2",
+        "pytest~=3.3",
```

Comment:
This is needed because we now have some tests that use the `caplog` fixture, which is not available in 3.2